MekHQ/src/mekhq/campaign/mission/ScenarioObjectiveProcessor.java

```diff
@@ -341,12 +337,21 @@ public String processObjective(ScenarioObjective objective, int qualifyingUnitCo
     }
 
     /**
-     * Processes an individual objective effect.
-     * @param effect
-     * @param scaleFactor If it's scaled, how much to scale it by
-     * @param tracker
+     * This method processes individual objective effects based on their type.
```

Comment:
what does it means "processes" in this context? It transform? return another value? derives some information?